src/main/java/com/pnu/dev/shpaltaif/service/PostServiceImpl.java

```diff
@@ -46,10 +48,15 @@ public PostServiceImpl(PostRepository postRepository, CategoryRepository categor
     public Page<Post> findAll(PostsPublicFilter postsPublicFilter, Pageable pageable) {
 
         Specification<Post> specification = postSpecificationBuilder.buildPostSpecification(postsPublicFilter);
-
         return postRepository.findAll(specification, pageable);
     }
 
+    @Override
+    public List<Post> findByTitleContains(String title, int size) {
+        Pageable pageable = PageRequest.of(0, size);
+        return postRepository.findAllByTitleContainsAndActiveTrue(title.trim(), pageable);
```

Comment:
I believe it can be `findTop5ByTitleContainsAndActiveTrue` and no need in pageable